src/platform/Linux/bluez/ChipDeviceScanner.h

```diff
@@ -44,6 +44,9 @@ class ChipDeviceScannerDelegate
 
     // Called when a scan was completed (stopped or timed out)
     virtual void OnScanComplete() = 0;
+
+    // Call on scan error
```

Comment:
Comment seems obvious.

Is the `OnScanComplete` comment above still valid: is the stop and timeout not an error?
Maybe comment here these special cases: are there cases when both complete and error may be called? Or are they mutually exclusive?

Reply:
Yes, this comment is still valid. This function in called when the scan is stopped or timed out (it is called with `OnScanError`).